addons/binding/org.openhab.binding.gpstracker/src/main/java/org/openhab/binding/gpstracker/internal/provider/gpslogger/GPSLoggerCallbackServlet.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.gpstracker.internal.provider.gpslogger;
+
+
+import org.eclipse.smarthome.core.thing.ThingRegistry;
+import org.openhab.binding.gpstracker.internal.discovery.TrackerDiscoveryService;
+import org.openhab.binding.gpstracker.internal.provider.AbstractCallbackServlet;
+import org.osgi.service.http.HttpService;
+
+/**
+ * Callback servlet for GPSLogger
+ *
+ * @author Gabor Bicskei - - Initial contribution and API
```

Comment:
Just a side note because I recognized this while reading through your code: This should be `@author Gabor Bicskei - Initial contribution` only.